app/scripts/lib/fxa-client.js

```diff
@@ -112,6 +112,16 @@ define(function (require, exports, module) {
           });
     },
 
+    checkAccountByEmailExists: function (email) {
+      return this._getClient()
+        .then(function (client) {
+          return client.accountStatus({email:email})
```

Comment:
It looks like my above comment could be applied to all the methods in the file. Ignore this suggestion then, stay consistent.